lib/active_record/connection_adapters/mysql2rgeo/spatial_table_definition.rb

```diff
@@ -22,6 +22,10 @@ def new_column_definition(name, type, **options)
 
           column
         end
+
+        def valid_column_definition_options
```

Comment:
Required by this change to ActiveRecord: https://github.com/rails/rails/pull/46178/files#diff-fac852e063eee718c2af44dec6f80b1b46f52f8aae7936e5eea6b2eab08cf2f7R590